share/wake/html/vis_arrow.js

```diff
@@ -0,0 +1,247 @@
+/**
+ * timeline-arrows
+ * https://github.com/javdome/timeline-arrows
+ *
+ * Class to easily draw lines to connect items in the vis Timeline module.
```

Comment:
We need to clarify what the license of this file is and include a reference to it into LICENSE in the repository root.

Reply:
Nevermind! It's APACHE-2, which is what wake is! So no need to call this file out.